server/config.ts

```diff
@@ -79,10 +79,10 @@ export default {
     reporting: {
       url: get('REPORTING_API_URL', 'http://127.0.0.1:3002', requiredInProduction),
       timeout: {
-        response: Number(get('REPORTING_API_TIMEOUT_RESPONSE', 10000)),
-        deadline: Number(get('REPORTING_API_TIMEOUT_DEADLINE', 10000)),
+        response: Number(get('REPORTING_API_TIMEOUT_RESPONSE', 60000)),
```

Comment:
Is the REPORTING_API_TIMEOUT_RESPONSE already defined somewhere in the environments or in the configuration in the code?
Or is it undefined and it will fall back to these default values?

Reply:
It's currently undefined in the environments.